example/cmd/PasswordGenerator.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/bitfield/qrand"
+)
+
+func main() {
+
+	var random = rand.New(qrand.NewSource())
+
+	// nchars stores the password length
+	var nchars = 15
+	charSet := []byte{'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'}
+	var password []byte
```

Comment:
And since it's a string we want, we may as well start with a string and append runes to it, instead of starting with a `[]byte` and appending bytes to it. After all, as we've established in Code Club seminars, a _really_ secure password generator would produce Unicode code points (up to 4 bytes each).